packages/fxa-settings/src/components/ButtonDownloadRecoveryKey/index.tsx

```diff
@@ -11,14 +11,15 @@ interface ButtonDownloadRecoveryKeyProps {
   navigateForward?: () => void;
   recoveryKeyValue: string;
   viewName: string;
+  keyWithContext?: boolean;
 }
 
 export const ButtonDownloadRecoveryKey = ({
   navigateForward,
   recoveryKeyValue,
   viewName,
+  keyWithContext = false,
```

Comment:
Is `Context` here just referring to the text around the key to give it context? It's just temporary so not a big deal but you might name it something else for clarity, the word `Context` is pretty loaded in the React world😬

Reply:
😬 good point, will rename. This is unrelated to React context